### PR TITLE
Upgrade drizzle and libsql

### PR DESCRIPTION
**Pull request by tom-sherman**

### CODE REVIEW

File: packages/frontpage/drizzle.config.ts [hunk 1]
```diff
@@ -12,11 +12,10 @@ if (!URL || !AUTH_TOKEN) {
 }
 
 export default defineConfig({
+  dialect: "turso",
```
Comment (damiensedgwick) [inline packages/frontpage/drizzle.config.ts:15]: Is this correct for a dialect?

File: packages/frontpage/drizzle.config.ts [hunk 2]
```diff
@@ -12,11 +12,10 @@ if (!URL || !AUTH_TOKEN) {
...
   schema: "./lib/schema.ts",
-  out: "./drizzle",
-  dialect: "sqlite",
-  driver: "turso",
```
Comment (damiensedgwick) [inline packages/frontpage/drizzle.config.ts:18]: Can we omit the driver?

File: packages/frontpage/drizzle.config.ts (hunk 1 above)
Comment (tom-sherman) [reply packages/frontpage/drizzle.config.ts:15]: https://orm.drizzle.team/docs/drizzle-config-file#dialect

File: packages/frontpage/drizzle.config.ts (hunk 2 above)
Comment (tom-sherman) [reply packages/frontpage/drizzle.config.ts:18]: > Drizzle Kit automatically picks available database driver from your current project based on the provided dialect

https://orm.drizzle.team/docs/drizzle-config-file#driver

File: packages/frontpage/drizzle.config.ts (hunk 1 above)
Comment (damiensedgwick) [reply packages/frontpage/drizzle.config.ts:15]: Oh nice, I did a quick check to see if I could find it in the docs but missed it.

**Review by damiensedgwick — APPROVED** [review]

**Pull request merged**